### DISCUSSION TITLE
Update documents by filter (with query parameter)

### DISCUSSION THREAD

**Discussion by erycson** [meilisearch/product#618] [Feedback & Feature Proposal]
Be able to update document attributes matching the same set of filters.

If it were in SQL, it would be something like: UPDATE index SET timestamp=1676466858 WHERE user_id=1

**Comment by macraig** [comment]
Hey @erycson! Apologies for the delayed response 🙇  

We're currently working on a prototype to GET and DELETE documents by filter as a first step. You can join the conversation in https://github.com/meilisearch/product/discussions/284 if you're interested. If our prototype is successful we will consider adding the Update feature in the near future 😄
